### PR TITLE
Display organization page

### PR DESCRIPTION
**Pull request by frinyvonnick**
Related #164

### CODE REVIEW

File: src/components/list/listItem.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import './listItem.css'
 const ListItem = ({
   onSelect, info, title, subtitle,
 }) => (
-  <div className="list-item" onClick={onSelect} role="button">
+  <div className={`list-item ${onSelect ? 'clickable' : ''}`} onClick={onSelect} role="button">
```
Comment (bpetetot) [inline src/components/list/listItem.jsx]: You should use `classnames` to do that.
```js
import cn from 'classnames'
// ...
<div className={cn('list-item', { 'clickable' : onSelect})} onClick={onSelect} role="button">
```

File: src/store/listeners/reactions/organizations.js [hunk 2]
```diff
@@ -61,3 +64,26 @@ export const fetchOrganizerOrganizations = reaction(async (action, store) => {
...
+    organization = { id: organizationId, ...ref.data() }
+    if (ref.exists) {
+      await store.data.organizations.add(organization)
+    }
+  }
+
+  const result = await fetchOrganizationUsers(organizationId)
+  const users = result.docs.map(userRef => ({ id: userRef.id, ...userRef.data() }))
+
+  store.data.organizations.update(set(organization, 'users', users))
```
Comment (bpetetot) [inline src/store/listeners/reactions/organizations.js]: also add users in `store.data.users` then they will be in cache for potential other use in the app

File: src/screens/organizer/organization/page/organizationPage.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  <div className="organization-page">
+    <Titlebar className="organization-header" icon="fa fa-users" title={name} />
+    <List
+      className="organization-content"
+      array={users}
+      noResult="No users yet !"
+      renderRow={({ id, displayName, updateTimestamp }) => (
+        <ListItem
+          key={id}
+          title={displayName}
```
Comment (bpetetot) [inline src/screens/organizer/organization/page/organizationPage.jsx]: Can you use the `Avatar` component here ?
```js
import Avatar from 'components/avatar'
// ...
title={<Avatar displayName={displayName} photoURL={photoURL} />}
```

**Review by bpetetot — CHANGES_REQUESTED** [review]

File: src/components/list/listItem.jsx (hunk 1 above)
Comment (frinyvonnick) [reply src/components/list/listItem.jsx]: My bad, I searched for `classNames` in the project to see if it was available. I didn't want to add a dependency 😂

**Review by bpetetot — APPROVED** [review]
👌

**Pull request merged**